pkg/proto/v0/accounts.pb.micro_test.go

```diff
@@ -423,6 +438,129 @@ func TestCreateAccountInvalidUserName(t *testing.T) {
 	cleanUp(t)
 }
 
+func TestUpdateAccount(t *testing.T) {
+	_, _ = createAccount(t, "user1")
+
+	tests := []struct {
+		name        string
+		userAccount *proto.Account
+	}{
+		{
+			"Update user (demonstration of updatable fields)",
+			&proto.Account{
+				DisplayName:                 "Alice Hansen",
+				PreferredName:               "Wonderful Alice",
+				OnPremisesDistinguishedName: "Alice",
+				UidNumber:                   20010,
+				GidNumber:                   30001,
+				Mail:                        "alice@example.com",
+			},
+		},
+		{
+			"Update user with unicode data",
+			&proto.Account{
+				DisplayName:                 "एलिस हेन्सेन",
+				PreferredName:               "अद्भुत एलिस",
+				OnPremisesDistinguishedName: "एलिस",
+				UidNumber:                   20010,
+				GidNumber:                   30001,
+				Mail:                        "एलिस@उदाहरण.com",
+			},
+		},
```

Comment:
You can also try more examples with different update values such as empty string, numbers, random string. Some of those might not work.